src/app/data/token-option-group.ts

```diff
@@ -11,6 +11,7 @@ export class TokenOptionGroup {
     private _description: string;
     private _isPublished: boolean;
     private _tokenOptions: TokenOption[];
+    public static TOKEN_OPTION_GROUP_MAX_SIZE = 17;
```

Comment:
I didn't do that because Angular's limited variable scope in HTML template makes it somehow complex, but you're right that use static variable instead of literals is definitely a better practice. I've change tooltip to use this static variable instead in the new commit.